src/main.cpp

```diff
@@ -250,7 +250,7 @@ int main(int argc, char* argv[])
 #endif // ! defined(QT_DEBUG)
         texts << QCoreApplication::translate("main", "Qt libraries %1 (compilation) %2 (runtime)\n",
              "%1 and %2 are version numbers").arg(QLatin1String(QT_VERSION_STR), qVersion());
-        texts << QCoreApplication::translate("main", "Copyright © 2008-2020  Mudlet developers\n");
```

Comment:
To help in this *next* year - can you drop in this about the line:
```
// PLACEMARKER: Date-stamp needing annual update
```
in to the source here... :smiling_imp: 

{`PLACEMARKER:` is one of the **not** predefined strings that I have in my Qt ToDi (not enable by default IIRC) plugin strings}:
![image](https://user-images.githubusercontent.com/6163092/103307159-523eb580-4a07-11eb-856f-dc253d810758.png)

Reply:
I don't think a placemaker is necessary because it took me 5mins to find these usages. Not worth littering the code for...

Reply:
Well, ***that*** is the whole purpose of those custom comment strings and the TODO Qt Creator plugin - there is filtering/sorting that can enable you to jump straight to them (which is why I have three at the places where one needs to add something for a new Lua function! :stuck_out_tongue_winking_eye:)

![image](https://user-images.githubusercontent.com/6163092/103377754-30613380-4ad8-11eb-8d2b-494fe4c2b7a9.png)

Reply:
Could you put documentation in wiki, so other coders can review and adapt? Thanks!

Reply:
:information_source:  See: https://wiki.mudlet.org/w/Using_Qt_Creator_TODO_plugin

Reply:
I think you're the only one who put them in and actually looks at them - but let's give it a go.